Apps/phdnsdb/dnsdb.json

```diff
@@ -1,413 +1,1504 @@
+                    "order": 3,
+                    "data_type": "string"
+                },
+                "time_last_before": {
+                    "description": "Record last seen before (epoch, relative seconds e.g. -31536000, or UTC timestamp e.g. 2021-01-05T12:06:02Z)",
+                    "order": 4,
+                    "data_type": "string"
+                },
+                "limit": {
+                    "description": "Max records to return",
+                    "order": 5,
+                    "data_type": "numeric",
+                    "default": 200
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.parameter.name",
+                    "data_type": "string",
+                    "contains": [
+                        "domain"
+                    ],
+                    "example_values": [
+                        "www.farsightsecurity.com"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.time_first_before",
+                    "data_type": "string",
+                    "example_values": [
+                        "-31536000",
+                        "2021-01-05T12:06:02Z"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.time_first_after",
+                    "data_type": "string",
+                    "example_values": [
+                        "-31536000",
+                        "2021-01-05T12:06:02Z"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.time_last_before",
+                    "data_type": "string",
+                    "example_values": [
+                        "-31536000",
+                        "2021-01-05T12:06:02Z"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.time_last_after",
+                    "data_type": "string",
+                    "example_values": [
+                        "-31536000",
+                        "2021-01-05T12:06:02Z"
+                    ]
+                },
+                {
+                    "data_path": "action_result.parameter.limit",
+                    "data_type": "numeric",
+                    "example_values": [
+                        200
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.count",
+                    "data_type": "numeric",
+                    "example_values": [
+                        100
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.rdata",
+                    "data_type": "string",
+                    "contains": [
+                        "domain"
```

Comment:
Please check for the output values for this datapath, if we are getting any IP values then put all the possible contains.

Reply:
For this action, only domains are retrieved in this data path. So keeping it as it is.